mappings/net/minecraft/server/rcon/RconBase.mapping

```diff
@@ -3,9 +3,15 @@ CLASS net/minecraft/class_3359 net/minecraft/server/rcon/RconBase
 	FIELD field_14424 description Ljava/lang/String;
 	FIELD field_14425 server Lnet/minecraft/class_2994;
 	FIELD field_14426 sockets Ljava/util/List;
+	FIELD field_14428 THREAD_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
 	FIELD field_14429 serverSockets Ljava/util/List;
+	FIELD field_14430 RCON_LOGGER Lorg/apache/logging/log4j/Logger;
```

Comment:
```suggestion
	FIELD field_14430 LOGGER Lorg/apache/logging/log4j/Logger;
```
It's a static field, only LOGGER should suffice

Reply:
When I tried logger, it said the name of the field would conflict with the logger in `RconClient`

Reply:
It's a known Enigma bug, the only solution right now is to edit the name in the mapping file directly. Static fields or methods don't conflict with parent/child classes

Reply:
oh okay